src/modules/esl-event-listener/core/api.ts

```diff
@@ -81,3 +81,6 @@ export class EventUtils {
     return listeners;
   }
 }
+
+/** @deprecated alias for {@link ESLEventListener} */
```

Comment:
If I'm not mistaken:
/** @deprecated alias for {@link ESLEventUtils} */
export const EventUtils = ESLEventUtils;